src/Data/Simulator.elm

```diff
@@ -151,32 +157,29 @@ computeMaterialAndSpinningCo2Score ({ inputs } as simulator) =
                     | co2 =
                         case ( inputs.material.recycledProcess, inputs.recycledRatio ) of
                             ( Just recycledProcess, Just ratio ) ->
-                                step.mass
+                                step.outputMass
                                     |> Co2.ratioedForKg
                                         ( recycledProcess.climateChange
                                         , inputs.material.materialProcess.climateChange
                                         )
                                         ratio
 
                             _ ->
-                                step.mass
+                                step.outputMass
                                     |> Co2.forKg inputs.material.materialProcess.climateChange
                 }
             )
 
 
 computeWeavingKnittingCo2Score : Simulator -> Simulator
-computeWeavingKnittingCo2Score ({ inputs, lifeCycle } as simulator) =
+computeWeavingKnittingCo2Score ({ inputs } as simulator) =
     simulator
         |> updateLifeCycleStep Step.WeavingKnitting
             (\({ country } as step) ->
                 let
                     { kwh, co2 } =
-                        -- NOTE: knitted elec is computed against previous step mass,
-                        -- weaved elec is computed against current step mass
```

Comment:
Special attention here, as I believe the comment was wrong (computed against FOLLOWING step)?